package.json

```diff
@@ -63,7 +63,7 @@
     "sinon": "^7.0.0",
     "source-map-support": "^0.5.6",
     "ts-mockito": "^2.2.5",
-    "typescript": "~3.1.0"
+    "typescript": "^3.1.4"
```

Comment:
👋 That's generally a scary thing to do.  TypeScript likes to make compiler changes between semver minor releases, which can cause your program to just stop compiling :(  It's obviously up to you how you want to manage this, but using a `^` range specifier will cause breaks on master at some point for you.

Reply:
This may not be necessary -- the compilation failed on our benchmark. 

Just trying everything to get the benchmark working, and I'll go back from there.

Reply:
Done